tsk/util/Bitlocker/MetadataValueStretchKey.h

```diff
@@ -0,0 +1,36 @@
+/*
+ ** The Sleuth Kit
+ **
+ ** Brian Carrier [carrier <at> sleuthkit [dot] org]
+ ** Copyright (c) 2024 Sleuth Kit Labs, LLC. All Rights reserved
+ ** Copyright (c) 2010-2021 Brian Carrier.  All Rights reserved
+ **
+ ** This software is distributed under the Common Public License 1.0
+ */
+
+#pragma once
+
+#ifdef HAVE_LIBMBEDTLS
+
+#include "MetadataValue.h"
+
+#define BITLOCKER_STRETCH_KEY_SHA256_LEN 32
+#define BITLOCKER_STRETCH_KEY_SALT_LEN 16
+
+class MetadataValueStretchKey : public MetadataValue {
+public:
+	MetadataValueStretchKey(BITLOCKER_METADATA_VALUE_TYPE a_valueType, uint8_t* buf, size_t bufLen);
+
+	BITLOCKER_STATUS parseStretchKeyUsingPassword(uint8_t* passwordHash, size_t passwordHashLen, uint8_t* stretchKey, size_t stretchKeyLen);
+
+	~MetadataValueStretchKey();
+private:
+	BITLOCKER_STATUS generateStretchedKey(uint8_t* passwordHash, size_t passwordHashLen, uint8_t* salt, size_t saltLen, uint8_t* result, size_t resultLen);
+
+	const size_t headerLen = 20;
+	BITLOCKER_ENCRYPTION_TYPE encryptionType;
```

Comment:
Should we be consistent here and renamed all of these to start with m_. I was confused while reviewing thinking that some places should be freeing data but then realized after doing a search that they were member variables and not local variables. I don't have a strong opinion just seems to deviate from our standard naming convention?

Reply:
I think I got them all